packages/nuxt/src/runtime/composables.ts

```diff
@@ -1,4 +1,27 @@
 import { useNuxtApp } from '#app'
+import {
+  defineStore as _defineStore,
+  type Pinia,
+  type StoreGeneric,
+} from 'pinia'
 export * from 'pinia'
 
 export const usePinia = () => useNuxtApp().$pinia
+
+export const defineStore = (...args) => {
```

Comment:
Isn't this going to break the types? we can probably use `const defineStore: typeof _defineStore` or just cast the function as `as typeof `_defineStore`